### DISCUSSION TITLE
Initial Feedback

### DISCUSSION THREAD

**Discussion by chrisjsewell** [aiidateam/aiida-website#2] [General]
For initial feedback of the website

Note, the initial goal is to try to replicate as much as possible the current https://www.aiida.net/

**Comment by csadorf** [comment]
@chrisjsewell I think it looks very good overall, especially the start page.

Here is my feedback:

- [x] The "Recent Posts" tab in the left sidebar looks a little wonky. I am not sure how to improve it, but maybe you could try to use a constant-width date format?
- [x] The pop-up dropdown for more on the original page makes sense, the "More" page on the new website looks very rough. If you want to keep it as its own page I'd suggest to add some very brief descriptions to the linked pages.
- [ ] You should add yourself to the AiiDA team page. 😃

**Reply by chrisjsewell** [reply]
> The pop-up dropdown for more on the original page makes sense

yeh this is already added in pydata-sphinx-theme (for the not yet released v0.10) 👍 https://github.com/pydata/pydata-sphinx-theme/pull/754, https://pydata-sphinx-theme.readthedocs.io/en/latest/

**Comment by giovannipizzi** [comment]
Thanks Chris for the great work!

Here are my comments:

- [x] Add "Download" page I would say as the second one, between About and News), similar to the [previous one](https://www.aiida.net/download/): I would keep the first three sections, remove the list of releases that made a lot of sense while we were transitioning between 0.x and 1.x but is not useful anymore (link instead to the GitHub releases page), keep the note on the licence in that page (I saw you moved it elsewhere)
- [x] I didn't see a link to the Roadmap anymore (old page pointing to the GitHub wiki). Should this be added as part of the new Download page?
- [x] finalize move of all news (probably taken care by #21)
- [ ] ~~Since the events are now under "More", maybe at the top of the "News" page, should we have a link to the Events as well (maybe with the top events visible, similar to the homepage?)~~ CJS: not possible, since this page is auto-generated by ablog
- [ ] We should ensure that all pages have proper redirects.
  - [ ] Document on the README where and how to add redirects
  - [x] Add redirects for all news (see #21)
  - [x] Add redirects for all main pages (`download`, `events`, `team`, `features`, `mailing-list`, `news`, `science`, `testimonials`, `graphs`, `acknowledgements`)
  - [x] Add redirects for all "Blog posts" (currently [here](https://www.aiida.net/posts/). I think it's OK not to have a dedicated page. I saw you already converted most (if not all of them). Can you also just do a double check from that page that we didn't miss any, while preparing the redirects?
- Add the text "AiiDA is supported by a [number of partners and funding institutions." in the footnote as in the old website, linking to the acknowledgements page, if possible?


Final general questions on URLs and redirects: 
- [x] can the part `/en/latest/` of the URL be removed and only point to that? 
- [x] Can we have a custom 404 page instead of the RTD one?
- [x] And, more generally, are the redirects already enabled and how do they work? For instance, I see this `"news/first-aiida-release-0-4-0/": "news/posts/2015-02-27-aiida-040-release",` in the [legacy redirect](https://github.com/aiidateam/aiida-website/blob/main/docs/legacy_redirect.json) file. However both https://aiida-website.readthedocs.io/en/latest/news/first-aiida-release-0-4-0/ and https://aiida-website.readthedocs.io/news/first-aiida-release-0-4-0/ do not work, maybe they are not enabled?
